src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,12 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        final Lottery lottery = new Lottery();
+        Ball[] balls = {lottery.getRandomBall(), lottery.getRandomBall(),
+                lottery.getRandomBall(), lottery.getRandomBall()};
```

Comment:
1. storing balls is not neccesary, as we dont want to use them later. 
2. what if we would want to create 100500 balls? create balls in loop instead
```suggestion
```